build/ci/windows/sign.bat

```diff
@@ -0,0 +1,95 @@
+@echo off
+setlocal ENABLEDELAYEDEXPANSION
+ECHO "MuseScore sign"
+
+SET CERT_ENCRYPT_SECRET=""
+SET CERT_PASSWORD=""
+SET FILES_DIR=""
+SET FILE_PATH=""
+SET FILE_NAME=""
+
+SET SIGNTOOL="C:\Program Files (x86)\Windows Kits\10\bin\x64\signtool.exe"
+SET SIGNCERT="build\ci\windows\resources\musescore.p12"
+
+SET TIMESERVER1="http://timestamp.globalsign.com/scripts/timstamp.dll"
+SET TIMESERVER2="http://timestamp.sectigo.com"
+SET TIMESERVER3="http://tsa.starfieldtech.com"
+
+:GETOPTS
+IF /I "%1" == "--dir" SET FILES_DIR=%2& SHIFT
+IF /I "%1" == "--file" SET FILE_PATH=%2& SHIFT
+IF /I "%1" == "--name" SET FILE_NAME=%2& SHIFT
+IF /I "%1" == "--secret" SET CERT_ENCRYPT_SECRET=%2& SHIFT
+IF /I "%1" == "--pass" SET CERT_PASSWORD=%2& SHIFT
+SHIFT
+IF NOT "%1" == "" GOTO GETOPTS
+
+IF %FILES_DIR% == "" ( 
+    SET MODE="file"
+) ELSE (
+    SET MODE="dir"
+)
+
+ECHO "MODE: %MODE%"
+ECHO "FILES_DIR: %FILES_DIR%"
+ECHO "FILE_PATH: %FILE_PATH%"
+ECHO "FILE_NAME: %FILE_NAME%"
+
+ECHO "Install secure-file"
+WHERE /q secure-file
+IF ERRORLEVEL 1 ( choco install -y --ignore-checksums secure-file )
+
+ECHO "Encrupt sert"
```

Comment:
Typo... should be "Encrypt cert"  I guess ;-)

Reply:
Or rather "Decrypt cert", as that's what it seems to be doing

Reply:
Fixed, thanks!